tools/get_release_contributors.py

```diff
@@ -31,26 +36,28 @@ class UnexpectedGithubStatus(RuntimeError):
     """
     Exception which is raised when an API request fails.
     """
-    pass
```

Comment:
Not needed as the function call ends here already?

Reply:
Pretty much - you can define the class with just the docstring in Python